core/ledger/kvledger/pkg_test.go

```diff
@@ -23,12 +23,15 @@ import (
 	"strconv"
 	"testing"
 
+	xtestutil "github.com/hyperledger/fabric/extensions/testutil"
 	"github.com/spf13/viper"
 )
 
 type testEnv struct {
-	t    testing.TB
-	path string
+	t          testing.TB
+	path       string
+	cdbAddress string
+	cdbCleanup func()
```

Comment:
cdbCleanup should be renamed to 'cleanup'.

Reply:
done